sopel/irc/backends.py

```diff
@@ -260,12 +260,17 @@ class SSLAsynchatBackend(AsynchatBackend):
                             (default ``True``, for good reason)
     :param str ca_certs: filesystem path to a CA Certs file containing trusted
                          root certificates
+    :param str certfile: filesystem path to a certificate for SASL EXTERNAL
+                         authentication (CERTFP)
```

Comment:
This is internal, but should probably also refer to it primarily as a client certificate.